azure-pipelines.yml

```diff
@@ -200,27 +223,22 @@ stages:
               mavenOptions: '-Xmx4g'
       - job: IT
         displayName: IT modules
-        timeoutInMinutes: '120'
+        timeoutInMinutes: '180'
```

Comment:
Do you know why it takes longer for the Spark 3 tests?  Or this change is not necessary.

Reply:
This change might not fully be necessary, but helps with the case for when i would see the tests slightly run over the limit and then auto terminate.